cloud/aws/integrations-aws.tf

```diff
@@ -1,12 +1,42 @@
-resource "signalfx_aws_external_integration" "sfx_integration_external" {
-  name = "AWSIntegration${var.suffix == "" ? "" : "-${title(var.suffix)}"}"
+locals {
+  integration_name = "AWSIntegration${var.suffix == "" ? "" : "-${title(var.suffix)}"}"
+  open_aws_services = [
+    for service in data.signalfx_aws_services.aws_services.services.*.name:
```

Comment:
.*. splat expression is deprecated in HCL2

Reply:
done